Templates/MonoGame.Templates.VSExtension/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Version="1.0.0.1" Id="MonoGame.Templates.VSExtension.03a6f6d5-6ec0-45ff-a4f6-ec098d51464d" Language="en-US" Publisher="MonoGame" />
+    <Identity Version="3.8.2.0" Id="MonoGame.Templates.VSExtension.03a6f6d5-6ec0-45ff-a4f6-ec098d51464d" Language="en-US" Publisher="MonoGame" />
```

Comment:
This should get replaced at publush time

Reply:
See https://github.com/MonoGame/MonoGame/blob/develop/build/BuildTemplatesTasks/BuildVSTemplatesTask.cs#L17

Reply:
But it won't matter that I've updated it, as it looks like it just replaces whatever is there, right?

Reply:
correct, so ignore me I guess :D